src/3rd_party-static/CMakeLists.txt

```diff
@@ -32,8 +32,8 @@
 add_subdirectory(./jsoncpp)
 set(JSONCPP_INCLUDE_DIRECTORY ${JSONCPP_INCLUDE_DIRECTORY} PARENT_SCOPE)
 
-# --- MessageBroker
-add_subdirectory(./MessageBroker)
+# --- Message Broker
+add_subdirectory(./message_broker)
```

Comment:
@anosach-luxoft this change has been made in pre_5_0_0_modifications, and looks like conversion all 3-rd part libraries to have names in lower case